relay/src/main.rs

```diff
@@ -51,11 +53,13 @@
 //!  - [`process-event`]: Process a Sentry event payload.
 //!  - [`scrub-minidump`]: Scrub PII from a Minidump file.
 //!
-//! [relay documentation]: https://docs.sentry.io/meta/relay/
+//! [relay documentation]: https://docs.sentry.io/product/relay/
 //! [`relay-auth`]: ../relay_auth/index.html
 //! [`relay-cabi`]: ../relay_cabi/index.html
 //! [`relay-common`]: ../relay_common/index.html
 //! [`relay-config`]: ../relay_config/index.html
+//! [`relay-ffi`]: ../relay_ffi/index.html
+//! [`relay-ffi-macros`]: ../relay_ffi_macros/index.html
 //! [`relay-filter`]: ../relay_filter/index.html
 //! [`relay-general`]: ../relay_general/index.html
 //! [`relay-quotas`]: ../relay_quotas/index.html
```

Comment:
You likely couldn't convert these because `relay` doesn't depend on them directly. See https://github.com/rust-lang/rust/issues/74481.